package.json

```
@@ -82,6 +82,10 @@
     "configuration": {
       "title": "Metals",
       "properties": {
+        "metals.coursierLocation": {
```

```suggestion
        "metals.coursierPath": {
```

sounds a bit better.

Also, can you add a `markdownDescription` field? This is the documentation that is displayed in the settings UI.

---

Changed.